examples/sanitizer/sanitizer.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See LICENSE.TXT
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ * @file codegen.cpp
```

Comment:
please remove or update these file names in file header if you copy it from other files.

Reply:
done